readme.md

```diff
@@ -26,7 +26,8 @@ To move the window, hover over the edge of the window so the resize arrow appear
 
 ### Windows
 
-If you use Windows and know how to move the window without the top bar, please let us know so we can add it here. :) Simply comment on [the issue about it](https://github.com/jancborchardt/hyperminimal/issues/12).
+To move the window, make sure Hyper has focus then press _Alt+Space+M_, then use the _Arrow_ keys to move the window and _Enter_ to finalise the position.
```

Comment:
It would be great if you could use `<kbd>Alt</kbd>+<kbd>Space</kbd>+<kbd>M</kbd>` so it renders in keyboard key style. :) <kbd>Alt</kbd>+<kbd>Space</kbd>+<kbd>M</kbd>`

Same for the <kbd>↑</kbd><kbd>→</kbd><kbd>↓</kbd><kbd>←</kbd> arrow keys and <kbd>Enter</kbd>. :)